packages/block-editor/src/components/copy-handler/README.md

```diff
@@ -0,0 +1,42 @@
+# Copy Handler
+
+The `CopyHandler` component handles the copy, cut and paste events of its children blocks. It handles multiple selection of blocks as well.
+
+Concretely, it handles the display of success messages and takes care of copying the block to the clipboard using the [raw handling API](https://github.com/WordPress/gutenberg/tree/master/packages/blocks/src/api/raw-handling).
```

Comment:
> copying the block to the clipboard using the raw handling API

Actually, the raw handling API is only used with _paste_ events (`pasteHandler`).

In contrast, cut/copy actions require what is essentially the opposite of the raw handling API: the `serialize` function, which outputs HTML augmented with the HTML-comment demarcations to denote blocks.

Both `pasteHandler` and `serialize` come from the `@wordpress/blocks` module, which is sometimes known as the _blocks _API_.